drivers/virtualbox/virtualbox.go

```diff
@@ -288,26 +289,37 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	// let VBoxService do nice magic automounting (when it's used)
-	if err := vbm("guestproperty", "set", d.MachineName, "/VirtualBox/GuestAdd/SharedFolders/MountPrefix", "/"); err != nil {
-		return err
-	}
-	if err := vbm("guestproperty", "set", d.MachineName, "/VirtualBox/GuestAdd/SharedFolders/MountDir", "/"); err != nil {
-		return err
-	}
+	var shareMountDir, shareDir, username string
 
-	var shareName, shareDir string // TODO configurable at some point
 	switch runtime.GOOS {
 	case "windows":
-		shareName = "c/Users"
-		shareDir = "c:\\Users"
+		username = os.Getenv("USERNAME")
+		shareDir = "c:\\Users\\" + username
+		shareMountDir = "/c/Users"
 	case "darwin":
-		shareName = "Users"
-		shareDir = "/Users"
-		// TODO "linux"
+		username = os.Getenv("USER")
+		shareDir = path.Join("/Users", username)
+		shareMountDir = "/Users"
```

Comment:
If we need static compilation, we want something like https://github.com/docker/docker/blob/master/pkg/homedir/homedir.go.  If we don't need static compilation, then [`user.Current`](https://golang.org/pkg/os/user/#Current) is going to be the "proper" solution.